src/.babelrc

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
we had 2 babelrcs in this repo. It was ruining everything. Thanks @unorthodoxgeek !!